Appraisals

```diff
@@ -1,13 +1,5 @@
 # frozen_string_literal: true
 
-appraise "cose_head" do
-  gem "cose", git: "https://github.com/cedarcode/cose-ruby"
-end
-
-appraise "openssl_head" do
-  gem "openssl", git: "https://github.com/ruby/openssl"
-end
```

Comment:
I know this were allowed to fail before, but isn't it necessary to still test them anyway on `.github/workflows/build.yml`?

Reply:
Ideally we would want to keep them, but GitHub Actions don't support something like TravisCI `allow_failures` setting. People asking for it in https://github.com/actions/toolkit/issues/399.

While it's somewhat useful to test against the head of this dependencies and see whether there's an upcoming incompatibility I don't see it as crucial. So that's the reason still moving to GitHub Actions despite not having that nice-to-have feature.